frontend/src/action/auth-actions.js

```diff
@@ -1,37 +1,48 @@
 import superagent from 'superagent';
+import * as utils from '../lib/util';
 
-export const tokenSet = token => {
+/********************************************************************************
+*         Synchronous                                                           *
+********************************************************************************/
+
+export const login = token => {
   return {
-    type: 'TOKEN_SET',
+    type: 'LOGIN',
     payload: token,
   };
 };
 
-export const tokenDelete = () => {
+export const logout = () => {
+  utils.cookieDelete('chatToken');
```

Comment:
Cookies should including the name of the app/project in them. For instance "rugbyscout_chatToken" would have been a better choice here.